src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -3977,6 +4004,16 @@ public TransformEngineChain getTransformEngineChain()
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override
+    public REDBlock getPayloadBlock(ByteArrayBuffer baf)
```

Comment:
should this be called `getREDBlock` ? (same for the method it calls inside as well)?  'payload' seems generic but this seems to return RED specifically